src/common/Defs.h

```diff
@@ -31,38 +31,26 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #ifndef COMMON_DEFS_H_
 #define COMMON_DEFS_H_
 
-#define PRODUCT_NAME        "Unvanquished"
-/** Case, No spaces */
-#define PRODUCT_NAME_UPPER  "UNVANQUISHED"
-/** No case, No spaces */
-#define PRODUCT_NAME_LOWER  "unvanquished"
+#ifndef ENGINE_NAME
+#define ENGINE_NAME "Daemon Engine"
+#endif
 
-#define PRODUCT_VERSION     "0.52.1"
+#ifndef ENGINE_VERSION
```

Comment:
What's this `ifndef`?

Reply:
It means “If not defined”.

I can write `#if !defined(ENGINE_VERSION)` instead if you prefer.

I do those `ifndef` because this allow to override them at build time without patching the source.